contracts/interfaces/BurnableERC20.sol

```diff
@@ -19,8 +19,10 @@ pragma solidity ^0.4.24;
 
 
 import 'openzeppelin-solidity/contracts/token/ERC20/StandardBurnableToken.sol';
+import 'openzeppelin-solidity/contracts/token/ERC20/CappedToken.sol';
 import './IBurnableERC20.sol';
 
-// TODO: Inherit from white list
-contract BurnableERC20 is IBurnableERC20, StandardBurnableToken {
+//   constructor (uint256 _cap) CappedToken(_cap) public {}
+contract BurnableERC20 is StandardBurnableToken, CappedToken {
```

Comment:
Why remove the burnable interface? We need a standard burnable interface to be first inherited. You can leave capped, but again this is a general purpose tech, someone may want to use a different implementation instead of capped token.